spec/warden/jwt_auth/strategy_spec.rb

```diff
@@ -30,6 +30,36 @@
         expect(strategy).not_to be_valid
       end
     end
+
+    context "when issuer is configured" do
+      let(:token) { Warden::JWTAuth::TokenEncoder.new.call({issuer: issuer}) }
+      let(:env) { { 'HTTP_AUTHORIZATION' => "Bearer #{token}" } }
+      let(:issuer) { 'http://example.com' }
```

Comment:
Let's use the shared config at `spec/support/shared_contexts/configuration` if possible, just like with the `aud` claim

Reply:
added in 508aa17f15075a91b5ed3b3623a4a5892c7b37ff